pkg/snapshotters/annotations.go

```diff
@@ -74,6 +74,33 @@ func AppendInfoHandlerWrapper(ref string) func(f images.Handler) images.Handler
 	}
 }
 
+// TargetCredsStreamLabel is a label which contains a stream ID that provides credentials via that stream.
+const TargetCredsStreamLabel = "containerd.io/snapshot/creds-stream"
```

Comment:
This stream ID is ephemeral and shouldn't be store for long use. Is the intent to just fetch this once and store it?